packages/sei-cosmwasm/README.md

```diff
@@ -57,6 +57,37 @@ Currently, Sei Bindings support query and message support for the sei custom mod
             - Change the Admin of the Denom. Only the current admin can change the admin.
         - SetMetadata
             - Set the denom metadata of a factory denom. Only the current admin can set metadata.
+- EVM
+    - Query
+        - StaticCall
+            - TODO
+        - Erc20TransferPayload
```

Comment:
want to confirm these are all "Gets"

Reply:
yeah these are just helpers for callers to format payload. Alternatively they can also format payload by themselves by reading ERC20's ABI in rust